i18n/en.pot

```diff
@@ -624,6 +624,26 @@ msgstr ""
 msgid "General Settings"
 msgstr ""
 
+msgid "Data sets specific download sync settings"
+msgstr ""
+
+msgid ""
+"Applies only to the assigned data set. Data set specific settings will "
```

Comment:
Seems to be some inconsistency in use of "data set" and "dataset". Could be a good idea to agree on the terminology with the product manager.

Reply:
Changing "dataset" to "data set"